hota/Mods/HotA configurable objects/content/config/hotaAncientLamp.json

```diff
@@ -7,13 +7,14 @@
 			{
 				"name": "Ancient Lamp (allows you to recruit 4-7 Master Genie once)",
 				"rmg" : {
-					"value"		: 5000,
+					"value"		: 500,
```

Comment:
In documentation, value is 5000. https://heroes.thelazy.net/index.php/Ancient_Lamp

Reply:
Huh, I was sure that it was 500 in docs yesterday... Fixing...